docs/workflow/requirements/windows-requirements.md

```diff
@@ -36,7 +36,8 @@ Note that Visual Studio and the development tools described below are required,
 
 * It's recommended to use **Workloads** installation approach. The following are the minimum requirements:
   * **.NET Desktop Development** with all default components,
-  * **Desktop Development with C++** with all default components.
+  * **Desktop Development with C++** with all default components, plus:
+    * *Windows 10 SDK (10.0.20348)* or newer, instead of version 10.0.19041
```

Comment:
```suggestion
  * **Desktop Development with C++** with all default components.
```
This should not be necessary. I do not think that Desktop Development with C++ in Visual Studio 2022 17.8+ installs this several years old SDK by default. The SDK installed by Visual Studio 2022 17.8+ should be new enough for us. (I am not able check 17.8, but the default in 17.9 that I am on is 10.0.22621.)

Reply:
I wonder if my machine was picking up some old artifact then. I removed all VS installs from my machine and then chose to reinstall VS 2022 Preview and version 10.0.19041 was checked and the others weren't. I'll revert the changes to that file and keep just the change to `.vsconfig`.

Thanks.